### PR TITLE
Default to link-dest on large-file backups

### PR DESCRIPTION
**Pull request by spads-spads**
We don't use attic for large file backups, and find that link-dest makes a much better default for these.

### CODE REVIEW

**Comment by rfinnie** [comment]
Leaving snapshot_mode null unless explicitly set by the source was deliberate in my original schema, and I don't think the API should be programatically setting it.  (A source's in-db config is basically just a copy of what the agent is sending to it.)

The variable handling of what snapshot mode is picked is already part of turku-storage `process_ping()` (which is way too long, ugh; check line ~173).  It currently uses mode "none" if either `large_rotating_files` or `large_modifying_files` are set, if no source `snapshot_mode` is defined.

Switching turku-storage's handling of `large_rotating_files` to link-dest makes sense (but only when combined with --fuzzy from your other PR, which seems fine).  However, in my mind `large_modifying_files` was envisioned for a situation where, say, you have a source with a 100GB daily database dump.  You probably don't want to default to any sort of snapshotting for that (unless you have lots of disk, in which case you could explicitly set `snapshot_mode: link_dest` on the source).

In summary, I think:

- This logic shouldn't be in turku-api
- turku-storage's ping handling should change `snapshot_mode = "none"` to `snapshot_mode = "link-dest"` for `large_rotating_files`
- `large_modifying_files` logic shouldn't be changed from default `snapshot_mode = "none"`
- rfinnie/turku-storage#1 should be merged regadless

(Oh, and BTW, attic mode was removed long, long ago, and everyone was happy)
